### ISSUE TITLE
`adoptium-ca-certificates` not in Debian Bookworm repository

### ISSUE THREAD

**Issue by syhe** [adoptium/adoptium-support#816]
### Please provide a brief summary of the bug

The package repository at https://packages.adoptium.net/artifactory/deb/ does not provide `adoptium-ca-certificates` for Debian Bookworm.
Other than that, it works really nice. Thank you for maintaining this distribution!

I realize that Bookworm is only going to be released as stable in three days, but I thought I'd ask beforehand, if you perhaps wanted to get things ready for release.

### Please provide steps to reproduce where possible

```Dockerfile
FROM debian:bookworm-slim
RUN set -eux; \
  apt-get update; \
  apt-get install -y --no-install-recommends \
    wget \
    apt-transport-https \
    ca-certificates
RUN mkdir -p /etc/apt/keyrings; \
  wget -O - https://packages.adoptium.net/artifactory/api/gpg/key/public | tee /etc/apt/keyrings/adoptium.asc; \
  echo "deb [signed-by=/etc/apt/keyrings/adoptium.asc] https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" | tee /etc/apt/sources.list.d/adoptium.list;
RUN set -eux; \
  apt-get update; \
  apt-get install -y --no-install-recommends \
    temurin-8-jdk
```

### Expected Results

`temurin-8-jdk` (and thus `adoptium-ca-certificates`) installs without an error message.

### Actual Results

(On Debian Bookworm, add the repository as per the [instructions](https://adoptium.net/installation/linux/), then try to install any Temurin version.)
```console
[…]
+ apt-get install -y --no-install-recommends temurin-8-jdk
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 temurin-8-jdk : Depends: adoptium-ca-certificates but it is not installable
E: Unable to correct problems, you have held broken packages.
```

### What Java Version are you using?

bash: java: command not found

### What is your operating system and platform?

Debian Bookworm

**Comment by Dragon-Knight** [comment]
Exactly the same problem when installing 'temurin-17-jdk'.
Debian 12x64.
What should I do to install jdk-17?

**Comment by sxa** [comment]
Yep this is broken. A temporary workaroudn is to use `bullseye` instead of `bookworm` in the repo file, which will allow it to work (The `.deb` packages are the same across all versions so it shouldn't cause ay problems)

**Comment by Dragon-Knight** [comment]
> Yep this is broken. A temporary workaround is to use `bullseye` instead of `bookworm` in the repo file, which will allow it to work (The `.deb` packages are the same across all versions so it shouldn't cause ay problems)

Thanks a lot!
PS: This installs the `adoptium-ca-certificates` package and can put back `bookworm` in `/etc/apt/sources.list.d/adoptium.list`.

**Comment by syhe** [comment]
@sxa How long does it usually take until changes like this propagate?
```console
% curl -s 'https://packages.adoptium.net/artifactory/deb/dists/bookworm/main/binary-amd64/Packages' | grep 'Package: ' | sort -u
Package: temurin-11-jdk
Package: temurin-11-jre
Package: temurin-17-jdk
Package: temurin-17-jre
Package: temurin-19-jre
Package: temurin-20-jdk
Package: temurin-20-jre
Package: temurin-8-jdk
Package: temurin-8-jre
```

**Comment by saghul** [comment]
Still seeing this error, has the package been uploaded? FWIW I'm getting it on aarch64.

**Comment by sxa** [comment]
Yeah this shouldn't be closed. There's an issue which @gdams is looking at for me to resolve this. The updates have not been fully propogated.

**Comment by sxa** [comment]
We've got this sorted now and it seems ok for me on Bookworm/aarch64 - let me know if you still get any problems!

**Comment by Hergam** [comment]
hello, it still not working:
![image](https://github.com/user-attachments/assets/cae7307d-ae1f-4112-8fc9-e43e0e8774d0)

**Comment by karianna** [comment]
@sxa could this be a arm64 vs aarch64 labelling issue?

**Comment by sxa** [comment]
@karianna @Hergam I've just run with the docker file from the description of this issue and it installed temurin-8-jdk with the adoptium-ca-certificates without problems inside the bookworm container on an Ubuntu 22.04/aarch64 host. Can you clarify what you did before running the apt-get install, and whether you were also on Debian bookworm?
